presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -105,6 +105,15 @@ General Aggregate Functions
 
     Returns ``n`` smallest values of all input values of ``x``.
 
+.. function:: reduce_agg(inputValue T, initialState S, inputFunction(S, T, S), combineFunction(S, S, S)) -> S
+
+    Returns a single value reduced from input values. ``inputFunction`` and ``combineFunction`` will be invoked.
+    ``inputFunction`` takes the current state (initially ``initialState``) and the input value, and returns the new state.
```

Comment:
does it mean `reduce_agg` runs on single node?
the wording suggests so; but existence of `combineFunction` suggests it is parallel

Reply:
No it runs in parallel. Any thoughts about how to revise the wording? -- Since when I read it, I didn't see it will run on single node, but I am clearly biased...

Reply:
@findepi, I don't think the name `combineFunction` implies parallelism.